muxers/mplex/Cargo.toml

```diff
@@ -18,7 +18,7 @@ libp2p-core = { version = "0.36.0", path = "../../core", default-features = fals
 log = "0.4"
 nohash-hasher = "0.2"
 parking_lot = "0.12"
-rand = "0.7"
+rand = "0.8"
```

Comment:
Given that this is a `dependency` and not a `dev-dependency` and given that the current `libp2p-mplex` version (`v0.36.0`) is released, this pull request needs to bump the patch version of `libp2p-mplex` to `v0.36.1`.

Please do the following:

- Bump version in `muxers/mplex/Cargo.toml`
- Add entry to `muxers/mplex/CHANGELOG.md
- Update `libp2p-mplex` in `Cargo.toml`
- Add changelog entry in `CHANGELOG.md`